draft-ietf-sipcore-sip-push.xml

```diff
@@ -310,29 +337,25 @@ align="center"><artwork>
           a REGISTER request prior to 120 seconds before the registration expires, the proxy MAY request a
           push notification towards the UA, to trigger the UA to send a REGISTER request.
         </t>
-        <t>
-          NOTE: In some cases the UA might be able to use a non-push mechanism (e.g., a timer) to awake and send 
-          re-registration REGISTER requests. Such REGISTER request will update the registration expiration timer, 
-          and the proxy does not need to request a push notification towards the UA in order to awake the UA. The
-          proxy will still request a push notification towards the UA when the proxy receives a SIP request addressed
-          towards the UA (<xref target="section.proxy.req.oth"></xref>). This allows the UA to e.g., use timers for sending 
-          re-registration REGISTER requests, but to be suspended (in order to save battery resources etc) between sending 
-          the REGISTER requests and use push notification to awaken the UA to process incoming calls.
-        </t>
       </section>
       <section anchor="section.proxy.req" title="SIP Request">
-      <section anchor="section.proxy.req.reg" title="REGISTER Request">
+      <section anchor="section.proxy.req.reg" title="REGISTER">
+        <t>
+          The procedures in this section apply when the SIP proxy receives a SIP REGISTER request (initial REGISTER request 
+          for a registration, or a re-registration REGISTER request) that contains a pn-provider SIP URI parameter identifying a type of PNS. 
+          If the proxy receives a REGISTER request that does not contain a pn-provider SIP URI parameter, or that removes the registration, 
+          the proxy MUST NOT request push notifications towards the UA associated with the REGISTER request, or perform any other procedures 
+          in this section.
+        </t>
         <t>
-          The procedures in this section apply when the SIP proxy receives a SIP REGISTER request (initial REGISTER request
-          for a registration, or a re-registration REGISTER request) that contains a pn-provider SIP URI parameter identifying a PNS.
-          If the proxy receives a REGISTER request that does not contain a pn-provider SIP URI parameter, or that removes the
-          registration, the proxy MUST NOT request push notifications towards the UA associated with the REGISTER request, or perform 
-          any other procedures in this section.
+          NOTE: As described in <xref target="section.ua.que"></xref>, a SIP UA might send a REGISTER request without including a pn-prid SIP URI parameter, 
+          in order to retrieve push notification capabilities from the network before it requests the proxy to request push notifications towards the UA. 
+          The procedures in this section apply to both the case when the REGISTER request contains a pn-prid SIP URI parameter, and when it does not.
```

Comment:
My idea was that the proxy encodes the information in the same way. The only difference is that, in case of query, the proxy will never request a push notification towards the UA.